test/statsd_instrumentation_test.rb

```diff
@@ -345,7 +345,7 @@ def test_statsd_count_with_injected_client
     client = StatsD::Instrument::Client.new(prefix: "prefix")
 
     ActiveMerchant::Gateway.statsd_count(:ssl_post, "ActiveMerchant.Gateway.ssl_post", client: client)
-    assert_statsd_increment("prefix.ActiveMerchant.Gateway.ssl_post", client: client) do
+    assert_statsd_increment("ActiveMerchant.Gateway.ssl_post", client: client) do
```

Comment:
I assume this changed because now we infer the `prefix` from the `client` that was passed in?

Reply:
Yeah it was a bug in the original test